packages/discovery-provider/src/api/v1/users.py

```diff
@@ -3007,3 +3011,160 @@ def get(self, id, authed_user_id):
         muted_users = get_muted_users(decoded_id)
         muted_users = list(map(extend_user, muted_users))
         return success_response(muted_users)
+
+
+encrypted_email_response = make_response(
+    "encrypted_email_response", ns, fields.List(fields.Nested(encrypted_email))
+)
+
+USER_ENCRYPTED_EMAILS_ROUTE = "/<string:id>/encrypted_emails"
+
+
+@ns.route(USER_ENCRYPTED_EMAILS_ROUTE)
+class UserEncryptedEmails(Resource):
+    @record_metrics
+    @ns.doc(
+        id="""Get Encrypted Emails""",
+        description="""Gets the encrypted emails where the given user is either the owner or has primary access""",
+        params={"id": "A User ID"},
+        responses={
+            200: "Success",
+            400: "Bad request",
+            401: "Unauthorized",
+            403: "Forbidden",
+            500: "Server error",
+        },
+    )
+    @ns.marshal_with(encrypted_email_response)
+    def get(self, id):
+        decoded_id = decode_with_abort(id, ns)
+
+        db = get_db_read_replica()
+        with db.scoped_session() as session:
```

Comment:
this is a really simple query and I think *probably* ok to be untested, but we typically do put queries inside src/queries as their own file so they can get their own tests, .e.g 

https://github.com/AudiusProject/audius-protocol/blob/main/packages/discovery-provider/src/queries/get_genre_metrics.py

https://github.com/AudiusProject/audius-protocol/blob/main/packages/discovery-provider/integration_tests/queries/test_get_genre_metrics.py